src/redux/books/book.js

```diff
@@ -34,25 +34,23 @@ const bookReducer = (state = startBooks, action) => {
         ...state, action.payload,
```

Comment:
- [OPTIONAL] Wonderful job so far. However, there is an issue with your add method. At this moment, all added books disappear after page refresh. I marked it optional because, it was not mentioned by the previous CR, but it is highly recommended to attend to it for it is part of core project requirement that must be fulfilled before your project can be approved. Kindly look into that and fix. Please, take a look.